### PR TITLE
feat(config,cpu,proofs): change cpu and batching defaults

### PR DESCRIPTION
**Pull request by justprosh**
## Description
default_min_batch_count=5
default_max_batch_count=30
default_max_proof_batch_size=128

default_system_cpu_count=1
default_cpus_range=1-total # we always leave 1 core to OS

## Checklist
- [ ] The code follows the project's coding conventions and style guidelines.
- [ ] All tests related to the changes have passed successfully.
- [ ] Documentation has been updated to reflect the changes (if applicable).
- [ ] All new and existing unit tests have passed.
- [ ] I have self-reviewed my code and ensured its quality.
- [ ] I have added/updated necessary comments to aid understanding.

## Reviewer Checklist
- [ ] Code has been reviewed for quality and adherence to [guidelines](https://doc.rust-lang.org/1.0.0/style/README.html).
- [ ] Tests have been reviewed and are sufficient to validate the changes.
- [ ] Documentation has been reviewed and is up to date.
- [ ] Any questions or concerns have been addressed.

### CODE REVIEW

File: crates/server-config/src/defaults.rs [hunk 1]
```diff
@@ -281,15 +274,15 @@ pub fn default_proof_poll_period() -> Duration {
...
 
 pub fn default_max_batch_count() -> usize {
-    4
+    30
 }
 
 pub fn default_max_proof_batch_size() -> usize {
-    2
+    128
 }
```
Comment (folex) [inline crates/server-config/src/defaults.rs:286]: Can you write some motivation for these numbers?

Comment (justprosh) [reply crates/server-config/src/defaults.rs:286]: Previous numbers had no motivation, these one are set by @mikevoronov request

File: crates/server-config/src/defaults.rs [hunk 2]
```diff
@@ -66,21 +66,14 @@ pub fn default_bootstrap_nodes() -> Vec<Multiaddr> {
...
 
 pub fn default_system_cpu_count() -> usize {
-    let total = num_cpus::get_physical();
-    match total {
-        x if x > 32 => 3,
-        x if x > 7 => 2,
-        _ => 1,
-    }
+    1
```
Comment (folex) [inline crates/server-config/src/defaults.rs:69]:
```suggestion
    // always use 1 core for Nox's and CCP's system threads
    // we use a simple constant here to make the behaviour as predictable as possible
    1
```

File: crates/server-config/src/defaults.rs (hunk 1 above)
Comment (folex) [reply crates/server-config/src/defaults.rs:288]: We can do better this time :)

File: crates/server-config/src/defaults.rs [hunk 3]
```diff
@@ -281,15 +277,15 @@ pub fn default_proof_poll_period() -> Duration {
 }
 
 pub fn default_min_batch_count() -> usize {
-    1
+    5
```
Comment (folex) [inline crates/server-config/src/defaults.rs:280]:
```suggestion
    // Wait for at least 5 proofs before sending the tx to reduce TPS
    5
```

File: crates/server-config/src/defaults.rs [hunk 4]
```diff
@@ -281,15 +277,15 @@ pub fn default_proof_poll_period() -> Duration {
...
 }
 
 pub fn default_max_batch_count() -> usize {
-    4
+    30
 }
 
 pub fn default_max_proof_batch_size() -> usize {
-    2
+    128
```
Comment (folex) [inline crates/server-config/src/defaults.rs:288]:
```suggestion
    // 128 is the average CU number on a single machine
    128
```

**Review by folex — APPROVED** [review]

**Pull request merged**